### PR TITLE
Add website to contributing readme

### PR DESCRIPTION
**Pull request by aliabd**
Long overdue but adds documentation on building the website, and contributing to docs and guides. 

@lappemic please feel free to add your thoughts and let me know if you have any suggestions! 

Closes: #8782
Closes: #5207

### CODE REVIEW

**Review by abidlabs — APPROVED** [review]
very nice!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -157,30 +157,93 @@ If you have made any significant visual changes to a component, we encourage you
...
-First, build the website:
+### Building The Website
+
+All of the website code lives in the `js/_website/` directory. 
+
+To start the website on dev mode simply cd into this directory and run: 
 
 ```
-pnpm build:cdn-local
+pnpm dev
```
Comment (abidlabs) [inline CONTRIBUTING.md:171]: You may need to do `pnpm i` first, right?

Comment (lappemic) [reply CONTRIBUTING.md:171]: For me worked pnpm dev

**Comment by lappemic** [comment]
This is great! LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (pngwn) [reply CONTRIBUTING.md:171]: If it hasn't been run before then pnpm i will be required. Can be run from anywhere.

**Review by pngwn — APPROVED** [review]
Looks great. Thanks @aliabd!

**Pull request merged**